pkg/api/validate_test.go

```diff
@@ -550,3 +550,69 @@ properties:
 		}
 	}
 }
+func TestValidHostnames(t *testing.T) {
```

Comment:
nit: newline before

Reply:
wouldn't it make sense to roll this into TestIsValidCloudAppHostname ?

Reply:
Sure